source/include/FreeRTOS_IP_Private.h

```diff
@@ -215,7 +215,7 @@ typedef struct IP_TASK_COMMANDS
 struct xPacketSummary
 {
     BaseType_t xIsIPv6;                          /**< pdTRUE for IPv6 packets. */
-    #if ( ipconfigUSE_IPv6 != 0 )
+    #if ipconfigUSE_IPV6
```

Comment:
@moninom1, although this PR is already merged, one minor remark: in all FreeRTOS sources, the following syntax is being used:
~~~c
#if ( configENABLE_MPU == 1 )
    ...
#endif
~~~
And you're right that we want to get rid of most of the `ipconfigUSE_IPv6` tests.

Thanks